command/CommandController.js

```diff
@@ -264,15 +274,44 @@ function getArticleMetadata(pocketArticle) {
     author = author ? `${author}, ${authors[auth].name}` : authors[auth].name;
   }
 
-  return {
-    item_id: pocketArticle.item_id,
-    sort_id: pocketArticle.sort_id,
-    resolved_url: pocketArticle.resolved_url,
-    title: pocketArticle.resolved_title,
-    author,
-    lengthMinutes,
-    imageURL: pocketArticle.top_image_url
-  };
+  let faviconPromise = getFavicons(pocketArticle.resolved_url);
```

Comment:
Let's protect this section with a request parameter will allow the Alexa skill to skip this additional metadata. We won't be using it there and the audio experience is more sensitive to long response times.

Reply:
Should I check the user-agent from the API call to decide if we should skip the favicon and publisher name calls?

Reply:
I don't know if we want to make assumptions based on user agent. I would lean toward something more explicit to give the client control over what they want to get back, so add a new body param like `completeData=1` or `extendedData=1` to trigger these additional queries.